docs/compute-scaling-factors.rst

```diff
@@ -0,0 +1,25 @@
+Compute Scaling factors
+-----------------------
+
+1- In the `Tools` menu, select `SF Calculator`.
+A new window will pop-up.
+
+2- The new window only has one activated field to enter a range of run
+numbers. Enter the following run range: `184975-184989` and click enter.
+
```

Comment:
It will be nice to have a screenshot here and point out that the loading could take more than a few seconds.